packages/test/src/integration-tests.ts

```diff
@@ -668,7 +668,6 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     const { BinaryChecksums, ...rest } = searchAttributes;
     t.deepEqual(rest, {
       CustomBoolField: [true],
-      CustomIntField: [], // clear
```

Comment:
This makes the client describe inconsistent with `workflowInfo().searchAttributes`, which still has empty array?

Reply:
I considered this but wasn't sure that's what we'd want since if you set an empty array in the workflow you'd probably expect to see an empty array when you read it back.
We need to do some more changes for search attributes to support standard persistence, let's tackle this when we do that batch of changes.